scenarios/berlin-v5.5-10pct/input/drt/berlin-drt-v5.5-10pct.config.xml

```diff
@@ -149,7 +149,7 @@
 				<param name="mode" value="walk" />
 			</parameterset>
 			<parameterset type="modeParams" >
-				<param name="marginalUtilityOfTraveling_util_hr" value="0.0" />
+				<param name="marginalUtilityOfTraveling_util_hr" value="-0.1" />
```

Comment:
Is it really utilityOfTravelling? I thought you changed the drt asc.

Reply:
gosh no :O you are right